source/02_euklido.rst

```diff
@@ -227,6 +245,12 @@ bendrąjį kartotinį:
     MBK := a * b div DBD(a, b);
   end;
 
+.. code-block:: unicode_cpp
+
+  long long MBK(long long a, long long b) {
+      return a * b / DBD(a,b);
+  }
+
 .. note::
 
   Svarbu nepamiršti, kad ``longint`` tipo kintamieji gali saugoti
```

Comment:
<this comment is not related to the change>

Correct, but by definition DBD is divisible by both a and b, so:

```
MBK(a, b) {
  return (a / DBD(a, b)) * b;
}
```

would be correct as long as the result of MBK fits into `long long` and hence work with a wider range of inputs (been bitten by this before)

Reply:
Thank you, fixed it!